.github/workflows/download-data.yml

```diff
@@ -20,4 +20,11 @@ jobs:
             - name: download schedule vs realtime
               uses: ./.github/actions/download-from-s3
               with:
-                file-path: schedule_vs_realtime_all_day_types_routes.json
\ No newline at end of file
+                file-path: schedule_vs_realtime_all_day_types_routes.json
+            - uses: oleksiyrudenko/gha-git-credentials@v2-latest
```

Comment:
I noticed that this generates a [warning](https://github.com/chihacknight/ghost-buses-frontend/actions/runs/7865233378) about updating to Node 20. I don't know if there is a later version for this action, but maybe line 15 could be changed to `actions/checkout@v4`?

Reply:
@dcjohnson24 I have upgraded checkout to version 4, but I am still getting a warning for `gha-git-credentials`. Unfortunately, there is not a newer version of this action. I can make a PR to that repo, but in the meantime I suggest living with the warning in our repo.

Reply:
@dcjohnson24 I made a PR in the github action to upgrade to version 20 [here](https://github.com/OleksiyRudenko/gha-git-credentials/pull/11).

Reply:
Thanks for looking into this and submitting the PR. I agree that we can live with the warning.